test/e2e/network/dns.go

```diff
@@ -68,7 +68,7 @@ var _ = SIGDescribe("DNS", func() {
 		validateDNSResults(f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
 
-	It("should resolve DNS of partial qualified names for the cluster ", func() {
+	It("should resolve DNS of partial qualified names for the cluster [LinuxOnly]", func() {
```

Comment:
It would be a good idea to comment why the tests are LinuxOnly, comment which should then be included in the test description if it becomes conformance.

For this test, Windows currently does not support resolving PQDNs.

Reply:
Done. Included as a comment.